pkg/ccl/backupccl/backup_planning.go

```diff
@@ -1095,6 +1094,35 @@ func backupPlanHook(
 	return fn, utilccl.BulkJobExecutionResultHeader, nil, false, nil
 }
 
+func protectTimestampForBackup(
+	ctx context.Context,
+	p sql.PlanHookState,
+	jr jobs.Record,
+	txn *kv.Txn,
+	resultsCh chan<- tree.Datums,
+	spans []roachpb.Span,
+	startTime, endTime hlc.Timestamp,
+	backupDetails jobspb.BackupDetails,
+) (*jobs.StartableJob, error) {
+	sj, err := p.ExecCfg().JobRegistry.CreateStartableJobWithTxn(ctx, jr, txn, resultsCh)
```

Comment:
Do we want this here? don't the two flavors use different job creation methods? I suspect we want just the `len(spans)... ` onwand and to pass the job ID